spec/requests/api/v1/npq_applications_spec.rb

```diff
@@ -377,6 +377,89 @@
         expect(parsed_response.dig("errors", 0, "detail")).to eql("Once rejected an application cannot change state")
       end
     end
+
+    context "when schedule identifier is in the params", with_feature_flags: { accept_npq_application_can_change_schedule: "active" } do
```

Comment:
pending which version this is for (v3 vs all versions) I think we might need to add this test to all request npq application versions